packages/world/mud.config.ts

```diff
@@ -91,19 +91,27 @@ export default mudConfig({
     KeysWithValue: {
       directory: "modules/keyswithvalue/tables",
       primaryKeys: {
+        sourceTable: "bytes32",
         valueHash: "bytes32",
       },
-      schema: {
-        keysWithValue: "bytes32[]", // For now only supports 1 key per value
+      schema: { length: "uint32", keys: "bytes32[]" }, // For now only indexes the first key of a primary key tuple
```

Comment:
We are already storing the length of the array in Store, why do we need to write the length as a filed too? Is it because it's blocked on getting the low level "getLength" function from StoreCore?

Reply:
yep, I haven't had the time to implement it yet

Reply:
I've updated this PR with those changes